src/Select.js

```diff
@@ -1141,7 +1142,7 @@ export default class Select extends Component<Props, State> {
       />
     );
   }
-  renderPlaceholderOrValue() {
+  renderPlaceholderOrValue(): ?PlaceholderOrValue {
```

Comment:
I think `Node | Node[]` is enough. `renderPlaceholderOrValue` is not used outside of `render`